references/documents.md

```diff
@@ -153,27 +146,25 @@ This `updateId` allows you to [track the current update](/references/updates.md)
 
 <RouteHighlighter method="PUT" route="/indexes/:uid/documents"/>
 
-Insert a list of documents or update them if they already exist based on [their unique identifiers](/guides/main_concepts/indexes.md#schema-definition).
+Add a list of documents or update them if they already.
 
-In case of an update, the old document will be only partially updated according to the fields in the request body. It will not be overwritten entirely.</br>
-To completely overwrite a document, check out the [add and replace documents route](/references/documents.md#add-or-replace-documents).
+If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **only the field of the new documents** will be changed in the old document. Thus, any fields not present in the new document remained unchanged.
```

Comment:
```suggestion
If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **only the fields of this new document** will be changed in the old document. Thus, any fields not present in the new document remain unchanged.
```

Reply:
I dont really understand the senteces. I guess that you try to describe the partial updating without mention it because I know MeiliSearch API (and because I can see it on the git diff).
I really think that the word "partial update" should stay, to be understood by most of the people. Anyone who used an API before know what a partial update is.
For those who don't know, indeed, it's ok to explain, and your explanation is welcome. But I would use more "field in the request body" than "fields of new document". I find it confusing.
Another point of view is needed :) @tpayet

Reply:
Do you like this more : 
`
If you send an already existing document (same [id](/guides/main_concepts/documents.md#identifier)) the old document will be only partially updated according to the fields of the new document. Thus, any fields not present in the new document are kept and remained unchanged.
`